lua/astrocommunity/pack/tailwindcss/init.lua

```diff
@@ -6,6 +6,30 @@ return {
     opts = function(_, opts)
       opts.ensure_installed = require("astrocore").list_insert_unique(opts.ensure_installed, { "tailwindcss" })
     end,
+    dependencies = {
+      {
+        "AstroNvim/astrolsp",
+        optional = true,
+        ---@type AstroLSPOpts
+        opts = {
+          handlers = {
+            tailwindcss = function()
+              require("lspconfig").tailwindcss.setup {
+                root_dir = function(fname)
+                  local root_pattern = require("lspconfig").util.root_pattern(
+                    "tailwind.config.cjs",
+                    "tailwind.config.js",
+                    "tailwind.config.ts",
+                    "postcss.config.js"
+                  )
+                  return root_pattern(fname)
+                end,
+              }
+            end,
+          },
+        },
```

Comment:
```suggestion
        opts = {
          config = {
            tailwindcss = {
              root_dir = function(fname)
                local root_pattern = require("lspconfig").util.root_pattern(
                  "tailwind.config.cjs",
                  "tailwind.config.js",
                  "tailwind.config.ts",
                  "postcss.config.js"
                )
                return root_pattern(fname)
              end,
            },
          },
        },
```

This should be done through the config table. This currently breaks the integration with AstroLSP like setting up our capabilities/on_attach as well as any configuration done by the user

Reply:
@mehalter thanks!